README.md

```
@@ -75,55 +75,194 @@ http://3.37.127.27
+erDiagram
+    users {
+        UUID id PK
+        VARCHAR email UK
+        VARCHAR nickname
+        VARCHAR password
+        BOOLEAN is_deleted
+        TIMESTAMPTZ deleted_at
+        TIMESTAMPTZ created_at
+        TIMESTAMPTZ updated_at
+    }
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)